test/errors/parsing/underscores.chpl

```diff
@@ -0,0 +1,34 @@
+// Underscoresa are disallowed in all named declarations except as components
+// of tuple decls and formals.
+var _ = 42;
+var (x, y) = (_, _);
+record _ {}
+class _ {}
+proc _(arg: int(?_)) {}
+interface _ {}
+enum _ {
+    _
+}
+
+// loops get a special nicer error message
+for _ in 1..10 {}
+foreach _ in 1..10 {}
+forall _ in 1..10 with (var _ = 42, + reduce _) {}
+coforall _ in 1..10 {}
+
+// Imports can't rename to '_'
+module A { var dummy: int; }
+import this.A as _;
+import this.A.{dummy as _};
+
+// Use statements can't rename limitations to '_'
+use A only dummy as _;
+
+// Underscores ARE allowed in tuple declarations...
+var (_, _) = (42, 42);
+// .. tuple assignments
+(x, y) = (42, 42);
```

Comment:
The test that underscores are allowed in tuple assignments doesn't include an underscore...

Reply:
Thank you for pointing this out! I will need to adjust the test.

Reply:
https://github.com/chapel-lang/chapel/pull/25303 adjusts the test to actually use an underscore 😄